### PR TITLE
Fix for issue #98. Hides comment form from unauthenticated users

### PR DESCRIPTION
**Pull request by pmeads**

### CODE REVIEW

File: app/views/comments/_comment.erb [hunk 1]
```diff
@@ -2,6 +2,10 @@
   <h3><%= comment.title %></h3>
   <p class="text-muted"><%= link_to comment.user.display_name, comment.user %>&nbsp;<%= l(comment.created_at, format: '%B, %d %Y %H:%M:%S') %></p>
   <blockquote>
-    <p>&nbsp;<%= comment.body %>&nbsp;<%= link_to 'Reply',  new_comment_path(comment.id) , :remote => true, :class => 'comment_reply_link' %></p>
+  	<p>&nbsp;<%= comment.body %>
+  	<% if current_user %>
+         &nbsp;<%= link_to 'Reply',  new_comment_path(comment.id) , :remote => true, :class => 'comment_reply_link' %>
+    <% end %>
```
Comment (chabgood) [inline app/views/comments/_comment.erb]: You can do link_to_if current_user .....

Comment (pmeads) [inline app/views/comments/_comment.erb]: Thanks Chris. Changed it

**Comment by chabgood** [comment]
:+1:

**Comment by ebollens** [comment]
@pmeads please update this branch against master so it can be merged through.

**Comment by ebollens** [comment]
This looks good to me.

**Pull request merged**